src/cli/exportGraph.js

```diff
@@ -78,41 +79,37 @@ export function makeExportGraph(
     }
 
     if (repoId == null) {
-      return die(std, "no repoId provided");
+      return die(std, "no repository ID provided");
     }
 
     const directory = Common.sourcecredDirectory();
     const registry = RepoIdRegistry.getRegistry(directory);
     if (RepoIdRegistry.getEntry(registry, repoId) == null) {
       const repoIdStr = repoIdToString(repoId);
-      std.err(`fatal: repoId ${repoIdStr} not loaded`);
-      std.err(`try running \`sourcecred load ${repoIdStr}\` first.`);
+      std.err(`fatal: repository ID ${repoIdStr} not loaded`);
+      std.err(`Try running \`sourcecred load ${repoIdStr}\` first.`);
       return 1;
     }
-
-    function promiseForAdapter(adapter: IAnalysisAdapter): Promise<Graph> {
-      const loadPromise = adapter.load(directory, NullUtil.get(repoId));
-      return new Promise((resolve, reject) => {
-        loadPromise
-          .then(resolve)
-          .catch((e) =>
-            reject(`plugin "${adapter.declaration().name}" errored: ${e}`)
-          );
-      });
+    async function graphForAdapter(adapter: IAnalysisAdapter): Promise<Graph> {
+      try {
+        return await adapter.load(directory, NullUtil.get(repoId));
+      } catch (e) {
+        throw Error(
```

Comment:
nit: `throw new Error` is idiomatic (though the forms are equivalent).

Reply:
idiom-ized